components/atom/input/src/Input/Component/_mixins.scss

```diff
@@ -0,0 +1,16 @@
+@mixin sui-atom-input-input {
+  background: $c-white;
+  border: 1px solid $c-gray-light;
+  box-sizing: border-box;
+  font-size: $fz-base;
+  height: $h-atom-input--m;
+  padding-left: $p-l;
+  padding-right: $p-l;
+  width: 100%;
+}
+
+@mixin sui-atom-input-input-focus {
+  border: solid 1px $c-primary;
+  box-shadow: 0 0 3px 0 $c-primary;
+  outline: 0 none;
+}
```

Comment:
In my humble opinion, I think that this could do it with an inheritance of styles instead with a mixin. To me, a mixin is like a function and in this case you are not passing any parameters to this function for this reason I think that you could do inheritance. Like %sui-atom-input-input-focus  {} and declare your sass inside. Where you need to apply this styles you would use @extends instead of @includes. Just my opinion, nice job! 👍